bundles/org.openhab.binding.broadlink/src/main/java/org/openhab/binding/broadlink/BroadlinkBindingConstants.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.broadlink;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.measure.Unit;
+import javax.measure.quantity.Dimensionless;
+import javax.measure.quantity.Power;
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link BroadlinkBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author John Marshall/Cato Sognen - Initial contribution
+ */
+@NonNullByDefault
+public class BroadlinkBindingConstants {
+
+    public static final ThingTypeUID THING_TYPE_RM2 = new ThingTypeUID("broadlink", "rm2");
+    public static final ThingTypeUID THING_TYPE_RM3 = new ThingTypeUID("broadlink", "rm3");
+    public static final ThingTypeUID THING_TYPE_RM3Q = new ThingTypeUID("broadlink", "rm3q");
+    public static final ThingTypeUID THING_TYPE_RM4_MINI = new ThingTypeUID("broadlink", "rm4mini");
+    public static final ThingTypeUID THING_TYPE_RM4_PRO = new ThingTypeUID("broadlink", "rm4pro");
+    public static final ThingTypeUID THING_TYPE_A1 = new ThingTypeUID("broadlink", "a1");
+    public static final ThingTypeUID THING_TYPE_MP1 = new ThingTypeUID("broadlink", "mp1");
+    public static final ThingTypeUID THING_TYPE_MP1_1K3S2U = new ThingTypeUID("broadlink", "mp1_1k3s2u");
```

Comment:
```suggestion
    public static final ThingTypeUID THING_TYPE_MP1_1K3S2U = new ThingTypeUID("broadlink", "mp1-1k3s2u");
```
Excellent that this is a constant, as it will make changing it far easier if all your code/classes uses THING_TYPE_MP1_1K3S2U in it.

https://www.openhab.org/docs/developer/guidelines.html#b-code-formatting-rules-style

lower-case-hyphen is what should be used for the thing type. I believe the others are fine to leave as is, but this uses the underscore, when the rule is to use hyphen. This kind of mix up really annoys the people that like textual config as a typo is easy to make and then hard to find so we have a rule. Try to have the thing type all lower and a single word if possible.